systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh

```diff
@@ -2,7 +2,7 @@
 
 NAMESPACE=$1
 
-KEYCLOAK_VERSION=11.0.0
+KEYCLOAK_VERSION=11.0.2
```

Comment:
Guess one last leftover :)

Reply:
This one was intentional... unless you want me to remove it :)